### ISSUE TITLE
[BUG] Flutter interface reports an error (the backend has been started and works normally on the web)

### ISSUE THREAD

**Issue by samtake** [openimsdk/openim-flutter-demo#127]
### OpenIM Server Version

3.8.1

### Operating System and CPU Architecture

macOS (AMD)

### Deployment Method

Source Code Deployment

### 2、啟動[openim-electron-demo](https://github.com/openimsdk/openim-electron-demo)後，接口正常。
![shopssnap](https://github.com/user-attachments/assets/e07600d2-7975-48b9-b431-34e933bb12c0)

### 3、啟動[open-im-flutter-demo](https://github.com/openimsdk/open-im-flutter-demo)後，登錄報錯
<img width="491" alt="nnnn" src="https://github.com/user-attachments/assets/4ea80c60-93c8-484c-b37c-4c34f61196bd">

```
D/EGL_emulation( 5353): app_time_stats: avg=507.36ms min=503.91ms max=510.81ms count=2
I/flutter ( 5353): ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────
I/flutter ( 5353): │ [http-request] [POST] http://localhost:10008/account/login
I/flutter ( 5353): │ Data: {
I/flutter ( 5353): │   "areaCode": "+86",
I/flutter ( 5353): │   "phoneNumber": "16624764738",
I/flutter ( 5353): │   "password": "500a32772f1bc403d3f3041225b97f61",
I/flutter ( 5353): │   "platform": 5
I/flutter ( 5353): │ }
I/flutter ( 5353): │ Headers: {
I/flutter ( 5353): │   "operationID": "1729148873189",
I/flutter ( 5353): │   "content-type": "application/json"
I/flutter ( 5353): │ }
I/flutter ( 5353): └──────────────────────────────────────────────────────────────────────────────────────────────────────────────
D/EGL_emulation( 5353): app_time_stats: avg=548.33ms min=481.97ms max=656.89ms count=3
I/flutter ( 5353): ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────
I/flutter ( 5353): │ [http-error] [POST] http://localhost:10008/account/login
I/flutter ( 5353): │ Message: The connection errored: Connection refused
I/flutter ( 5353): │ Headers: {
I/flutter ( 5353): │   "operationID": "1729148873189",
I/flutter ( 5353): │   "content-type": "application/json",
I/flutter ( 5353): │   "content-length": "105"
I/flutter ( 5353): │ }
I/flutter ( 5353): └──────────────────────────────────────────────────────────────────────────────────────────────────────────────
[OpenIM-App] ** e:接口：http://localhost:10008/account/login  信息：The connection errored: Connection refused s:, isError [false]
[OpenIM-App] ** login e: 接口：http://localhost:10008/account/login  信息：The connection errored: Connection refused , isError [false]
```

**Comment by std-s** [comment]
Try using IP.

**Comment by samtake** [comment]
Fixed, set IP to 10.0.2.2 on Android emulator to make request
